tests/components/unifiprotect/test_switch.py

```diff
@@ -333,24 +333,23 @@ async def test_switch_camera_simple(
     doorbell.__pydantic_fields__[description.ufp_set_method] = Mock(
```

Comment:
maybe in a followup PR, but should setting the `__pydantic_fields__` also be recplaced with `patch`?

Reply:
https://github.com/home-assistant/core/pull/159346